srcpkgs/powerline-go/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'powerline-go'
+pkgname=powerline-go
+version=1.17.0
+revision=1
+archs="noarch"
```

Comment:
This is definitely not noarch

Reply:
Just remove the line altogether

Reply:
i'm porting this package on void. i don't know more information. On homepage site not provides architecture information.

Reply:
If it's simple go, it can likely be built for all archs. `noarch` is for packages which don't have architecture specific stuff, such as pure python programs or fonts. You need to specify `archs=` only for cases where for some reason the program can't be built for/doesn't work on a specific architecture.

Reply:
Thanks for explanation. 
I didn't found architecture information, therefore, i suppose that this package is right everywhere.
I'm making simply porting from Aur repository and for this reason there is X86_64.
Here, there are more controls. 
I think that it's right.

Reply:
Yeah, Arch having a single target architecture makes things somewhat different from how we do them. Glad to help c: